.github/CODEOWNERS

```
@@ -1,41 +1,21 @@
 # Global Owners
-* @nickboldt @ibuziuk @l0rd @sparkoo
+* @l0rd @ibuziuk @vinokurig @nickboldt 
 
-# dockerfiles
-dockerfiles/** @nickboldt
-
-# assembly
-assembly/** @nickboldt
-assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/** @ibuziuk @nickboldt @themr0c
-
-# core
-core/** @nickboldt
+# che.properties (quasi-documentation)
+# Fionn Kelleher (@osslate) is the replacement for Fabrice, but he doesn't seem to have rights to this repo, so comment this out for now
+# assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/** @osslate 
 
 # workspace runtime infrastructure implementations
-infrastructures/** @amisevsk @nickboldt
+infrastructures/** @amisevsk 
 
 # multiuser che
-multiuser/api/**  @nickboldt
-multiuser/integration-tests/**  @mshaposhnik @nickboldt
-multiuser/keycloak/**  @mshaposhnik @nickboldt
-multiuser/machine-auth/**  @mshaposhnik @nickboldt
-multiuser/permission/**  @nickboldt
-multiuser/personal-account/**  @nickboldt
-multiuser/sql-schema/**  @mshaposhnik @nickboldt
+multiuser/**  @mshaposhnik
 
 # wsmaster
-wsmaster/** @nickboldt @vinokurig 
-wsmaster/che-core-api-factory/** @mshaposhnik @nickboldt
-wsmaster/che-core-api-factory-shared/**   @mshaposhnik @nickboldt
-wsmaster/che-core-api-workspace-activity/**   @mshaposhnik @nickboldt
-wsmaster/che-core-api-workspace/**  @mshaposhnik @nickboldt
+wsmaster/** @vinokurig @mshaposhnik
```

I do not think Maksim will be still owning wsmaster

---

me neither..:)

---

If you use the `suggestion` feature we can skip a step and collaborate faster. 
```suggestion
wsmaster/** @vinokurig 
```